ui/nuxeo-pagination-controls.js

```diff
@@ -42,18 +43,55 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
             display: block;
           }
 
+          .container {
+            @apply --layout-horizontal;
+            @apply --layout-center;
```

Comment:
To use this, we should try to import the correspondent files (despite it work because they are globally imported)
`import '@polymer/iron-flex-layout/iron-flex-layout.js';`